SeleniumTestXUnit/Tests/Features/Item/AddBelow.feature

```diff
@@ -0,0 +1,19 @@
+Feature: Add Note Below
+    As a logged in user, I want to be able to create a note below another
+
+
+    Background:
+        Given the user is logged in
+        And the user has existing notes
+
+    @acceptance
+    Scenario: Add below
+        Given the user wants to add a new note below an existing one
+        When the user hovers over the desired note
+        And clicks on the menu list
+        And clicks on the priority "Add item below" button
+        And an input box should be displayed
```

Comment:
This sounds like a validation but you are still on a When step here, checking whether the input box is enabled to start typing on it should be done behind scenes on the step implementation.